tests/test_db.py

```diff
@@ -66,3 +70,8 @@ def test_negative(self):
         with pytest.raises(Exception) as ex_info:
             db.psql("SELECT * FROM table ERROR")
         assert "Syntax Error" in str(ex_info.value)
+
+    def test_psql_cmd(self):
+        db = self.get_db()
+        res = db.psql_cmd('\\\dt')
+        assert res
```

Comment:
You are asserting "" string defined in line 39 above ^.  so probably you need assert res == '' to test this.

Reply:
Thanks, now I know where it gets the output.